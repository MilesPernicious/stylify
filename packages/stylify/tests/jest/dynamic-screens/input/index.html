<div class="
	color:red
	minw640px:color:blue
	minw1024px&&maxw1280:color:green
	minw1280px||landscape:color:purple
	minw1280px||landscape:hover:color:purple
">
</div>
