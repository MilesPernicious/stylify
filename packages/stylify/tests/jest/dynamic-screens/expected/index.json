{"mangleSelectors":false,"reconfigurable":true,"dev":true,"selectorsList":{"color:red":{"mangledSelector":"s0","processed":true},"minw640px:color:blue":{"mangledSelector":"s1","processed":true},"minw1024px&&maxw1280:color:green":{"mangledSelector":"s2","processed":true},"minw1280px||landscape:color:purple":{"mangledSelector":"s3","processed":true},"minw1280px||landscape:hover:color:purple":{"mangledSelector":"s4","processed":true}},"mangledSelectorsMap":{"s0":"color:red","s1":"minw640px:color:blue","s2":"minw1024px&&maxw1280:color:green","s3":"minw1280px||landscape:color:purple","s4":"minw1280px||landscape:hover:color:purple"},"cssTree":{"_":{"color:red":{"selectors":["color:red"],"properties":{"color":"red"}}},"(min-width: 640px)":{"minw640px:color:blue":{"selectors":["minw640px:color:blue"],"properties":{"color":"blue"}}},"(min-width: 1024px) and (max-width: 1280)":{"minw1024px&&maxw1280:color:green":{"selectors":["minw1024px&&maxw1280:color:green"],"properties":{"color":"green"}}},"(min-width: 1280px), (orientation: landscape)":{"minw1280px||landscape:color:purple":{"selectors":["minw1280px||landscape:color:purple"],"properties":{"color":"purple"}},"minw1280px||landscape:hover:color:purple":{"selectors":["minw1280px||landscape:hover:color:purple:hover"],"properties":{"color":"purple"}}}}}